frontend/styles/site.less

```diff
@@ -2225,6 +2226,18 @@ table {
 	}
 }
 
+#chart {
+    width: 1000px;
+    height: 600px;
```

Comment:
I almost hate to ask (and we can postpone the answer), but what happens to this on mobile?